server/src/diskdb.ts

```diff
@@ -71,13 +76,44 @@ export class Disk implements Repository {
     return projectId;
   }
 
-  public async get(projectId: string): Promise<DatabaseModel> {
+  public async get(projectId: string): Promise<ProjectModel> {
     const project = this.db.get(projectId);
 
-    if (!project) {
+    if (!project || !isProject(project)) {
       throw `Can't find project with id: ${projectId}`;
     }
 
     return project;
   }
+
+  public async getQuickChecklist(id: string): Promise<QuickChecklistModel> {
+    const quickChecklist = this.db.get(id);
+
+    if (!quickChecklist || isProject(quickChecklist)) {
```

Comment:
`isProject(quickChecklist)` doesn't really read right

Reply:
Fair, will rename it to as you suggested, isValidProject and isValidQuickChecklist.